src/components/index.jsx

```diff
@@ -28,19 +29,38 @@ const MainCard = styled.div`
+				how_to_apply: '<p>Click this link to apply: <a href="https://tubescience.com/were-hiring/?gh_jid=5039665002">https://tubescience.com/were-hiring/?gh_jid=5039665002</a></p>\n',
+				company_logo: 'https://jobs.github.com/rails/active_storage/blobs/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBbStXIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--b8a551a3c84317266c1f3dfe846ded212a933e24/TS%20Linkedin%20Logo.jpeg',
+			},
+		],
+	);
```

Comment:
Why do we need this object in here?

Reply:
the app is not running wthen state is `null` at first...so i added that as the default

Reply:
So show a loading state when it's `null`. If you don't wan ta loading state (which I think you do) make the default value `[]`.